airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -34,6 +35,146 @@
         }
     )
     .set_file_type("csv")
+    .set_expected_spec(
+        {
+            "documentationUrl": "https://docs.airbyte.com/integrations/sources/in_memory_files",
+            "connectionSpecification": {
+                "title": "InMemorySpec",
+                "description": "Used during spec; allows the developer to configure the cloud provider specific options\nthat are needed "
+                               "when users configure a file-based source.",
+                "type": "object",
+                "properties": {
+                    "streams": {
+                        "title": "The list of streams to sync",
+                        "description": "Streams defines the behavior for grouping files together that will be synced to the downstream "
```

Comment:
I think this description of streams is hard to understand for an end user. A suggested alternate phrasing: 

```
Each instance of this configuration defines which defines a <href rel="https://docs.airbyte.com/cloud/core-concepts#stream">stream</href>. Use this to define which files belong in the stream, their format, and how they should be parsed and validated. When sending data to warehouse destination such as Snowflake or BigQuery, each stream is a separate table. 
```